scalalib/src/ScalaModule.scala

```diff
@@ -86,35 +85,53 @@ trait ScalaModule extends JavaModule { outer =>
       publish.Artifact.fromDep(
         _: Dep,
         scalaVersion(),
-        mill.scalalib.api.Util.scalaBinaryVersion(scalaVersion()),
+        ZincWorkerUtil.scalaBinaryVersion(scalaVersion()),
         platformSuffix()
       )
     }
 
   /**
    * Allows you to make use of Scala compiler plugins from maven central
    */
-  def scalacPluginIvyDeps = T { Agg.empty[Dep] }
+  def scalacPluginIvyDeps: Target[Agg[Dep]] = T { Agg.empty[Dep] }
 
-  def scalaDocPluginIvyDeps = T { scalacPluginIvyDeps() }
+  def scalaDocPluginIvyDeps: Target[Agg[Dep]] = T { scalacPluginIvyDeps() }
 
   /**
    * Mandatory command-line options to pass to the Scala compiler
    * that shouldn't be removed by overriding `scalacOptions`
    */
-  protected def mandatoryScalacOptions = T { Seq.empty[String] }
+  protected def mandatoryScalacOptions: Target[Seq[String]] = T { Seq.empty[String] }
+
+  /**
+   * Scalac options to active the compiler plugins.
+   */
+  private def enablePluginScalacOptions: Target[Seq[String]] = T {
+    val resolvedJars = resolveDeps(scalacPluginIvyDeps.map(_.map(_.exclude("*" -> "*"))))()
+    resolvedJars.iterator.map(jar => s"-Xplugin:${jar.path}").toSeq
+  }
+
+  /**
+   * Scalac options to active the compiler plugins.
+   */
+  private def enableScalaDocPluginScalacOption: Target[Seq[String]] = T {
```

Comment:
This should be plural, right? It is a `Seq[String]`..

Reply:
Yes. Fixed.